spec/features/single_content_item_spec.rb

```diff
@@ -46,6 +57,18 @@
       expect(page).to have_content('200000')
     end
 
+    it 'renders a metric for satisfaction_score' do
+      expect(page).to have_content('25.5')
+    end
+
+    it 'renders the page title' do
+      expect(page).to have_content('Content Title')
```

Comment:
This test relies on the existence of `Content Title` in the whole page. It is better to use selectors:

```ruby
expect(page).to have_selector('css_selector', text: 'Content Title')
```

If not, we rely on extending the feature without having duplicated values.